zaza/openstack/utilities/openstack.py

```diff
@@ -238,7 +238,7 @@ async def _check_ca_present(model, ca_files):
             for unit in units:
                 try:
                     output = await unit.run('cat {}'.format(ca_file))
-                    contents = output.data.get('results').get('Stdout', '')
+                    contents = output.data.get('results', {}).get('Stdout', '')
```

Comment:
Hmm, I think the problem here is that this is invoking direct juju invocations rather than the zaza model. The zaza model has logic to deal with the async result differences between libjuju 2.x and 3.x, but this code doesn't have that.